src/PrestaShopBundle/Controller/Admin/Sell/Catalog/CartRuleController.php

```diff
@@ -24,13 +24,27 @@
  * International Registered Trademark & Property of PrestaShop SA
  */
 
+declare(strict_types=1);
+
 namespace PrestaShopBundle\Controller\Admin\Sell\Catalog;
 
```

Comment:
```suggestion
namespace PrestaShopBundle\Controller\Admin\Sell\Catalog;

```
It's not a new file : don't add `declare(strict_types=1);`